helm-chart/renku-gateway/templates/_helpers.tpl

```diff
@@ -41,13 +41,3 @@ https
 http
 {{- end -}}
 {{- end -}}
-
-{{/*
-Hack for calling templates in a fake scope (until this is solved https://github.com/helm/helm/issues/3920)
```

Comment:
All for removing this hack. Last time I checked this was still necessary, however I was using an oder version of helm. The [fix PR](https://github.com/helm/helm/pull/9957) was merged in helm 3.7. Should we let this hack live a little longer or force everybody to use helm 3.7 or later?

Reply:
Ah, I see that this hack fails linting of this chart on its own (ie in absence of renku which brings redis as dependency...). This raises the more fundamental question if this chart can be deployed standalone. If yes, we have to include redis as dependency. If no, it should probably be part of renku... 🤔

Reply:
What I don't want to is to force a user to specifically define the redis host in the global values. We have to be able to infer it automatically for renku deployments which include redis.

Reply:
@ableuler the user already has to define all the other values related to redis in the global portion anyhow. But we can resolve the host issue by using this expression for the host: `{{ .Values.global.redis.host | default (printf "%s-redis" .Release.Name) }}`. This works properly when left to default and paired with renku. The CI deployment for this PR succeeded with this change.

Reply:
... and I tested that I could launch a session.

Reply:
`.Values.global.redis.clientLabel` might be useful anyway as it could be very likely that an externally deployed redis instance requires all potential client pods to have some specific label.